src/org/sage_one_sample/sageone/SageoneData.java

```diff
@@ -0,0 +1,88 @@
+package org.sage_one_sample.sageone;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.net.HttpURLConnection;
+import org.json.JSONObject;
+import java.util.*;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+/**
+ * Servlet implementation class GetData
+ */
+@WebServlet("/SageoneData")
+public class SageoneData extends HttpServlet {
+	private static final long serialVersionUID = 1L;
+
+  /**
+  * @see HttpServlet#HttpServlet()
+  */
+  public SageoneData() {
+    super();
+    // TODO Auto-generated constructor stub
```

Comment:
I don't think you need this constructor here as all it does it call super(). If you do need it then you can just delete the TODO